### PR TITLE
chore: add logging example

### PR DESCRIPTION
**Pull request by lyl156**
It is PR for [issue#2238](https://github.com/open-telemetry/opentelemetry-go/issues/2238).

### CODE REVIEW

File: example/logging/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# OpenTelemetry Collector Traces Example
+This example is to just export OTLP to a collector running locally (via Docker) and print the trace data to stdout.
+# Running the code
+```
+sh build.sh  
+go run main.go
+```
+Now you can check the trace data generated by `main.go` on the terminal where you type `sh build.sh`.
```
Comment (hanyuancheung) [inline example/logging/README.md:8]: Pls fix the markdown format.

Comment (lyl156) [reply example/logging/README.md]: fixed

**Review by lyl156 — COMMENTED** [review]
CI is not clear now. 

Should I pull this and re-submit again?
Or is there something I can do to clear the CI?

**Comment by hanyuancheung** [comment]
> CI is not clear now.
> 
> Should I pull this and re-submit again? Or is there something I can do to clear the CI?

rerun `make precommit` command before committing the code.

**Comment by lyl156** [comment]
> > CI is not clear now.
> > Should I pull this and re-submit again? Or is there something I can do to clear the CI?
> 
> rerun `make precommit` command before committing the code.

**Comment by lyl156** [comment]
reopen it

**Review by MrAlias — CHANGES_REQUESTED** [review]
This PR adds an example of how to use the logging exporter for the collector. The linked issue this PR says it addresses is one that looks to refactor the existing collector example, not add another one with a different exporter. I do not see the value in accepting these changes, it does not add a unique example of how to use this project. I would recommend it is closed.

**Comment by dmathieu** [comment]
I think naming this example `logger` is misleading. It may look like the example is about using the library to do logging.
I also agree with @MrAlias that we don't need a new example if one is too complicated. The existing example should be simplified, or refactored to be more understandable (for example with steps of increasing complexity).

**Comment by MrAlias** [comment]
> I think naming this example `logger` is misleading. It may look like the example is about using the library to do logging. I also agree with @MrAlias that we don't need a new example if one is too complicated. The existing example should be simplified, or refactored to be more understandable (for example with steps of increasing complexity).

@lyl156 is this something you are willing to consider?

**Comment by lyl156** [comment]
> 
What I can do now is to replace the original with mine, because I don't understand your exact thoughts on this issue. 
If you can provide exact examples representing the result you want, I'd love to revise this pr.
If not, I will close this pr soon.

**Pull request closed without merging**